bones/bone.py

```diff
@@ -106,7 +106,7 @@ def __init__(self, descr="", defaultValue=None, required=False, params=None, mul
 		self.params = params or {}
 		self.multiple = multiple
 		self.indexed = indexed
-		self.defaultValue = defaultValue
+		self.defaultValue = [] if defaultValue is None and self.multiple else defaultValue
```

Comment:
And what about the cases [here](https://github.com/viur-framework/viur-core/blob/master/bones/bone.py#L397) and [here](https://github.com/viur-framework/viur-core/blob/master/bones/bone.py#L404)? My solution from #213 fixes all of them, and also cleans up the code.

Reply:
Thanks for pointing that out. Ive pushed a new commit that also handles the multi-language case.
With the new skeleton system in ViUR3, it should be guaranteed that a bone has always the correct type expected by the bone (even if being empty) - that's why stringBones initializes to an empty string, numericBones to 0 etc. This bug prevented this on bones with multiple or languages, as these will change the type that's stored in the skeleton.